bootstrap/sql/mysql/v003__create_db_connection_info.sql

```diff
@@ -106,6 +106,10 @@ ADD INDEX (deleted);
 UPDATE entity_relationship
 SET relation = 10 WHERE fromEntity = 'team' AND toEntity = 'user' AND relation = 0;
 
+-- Change "dashboard -- contains --> chart" relationship to "dashboard -- has --> chart" relationship
```

Comment:
Should this go into v004?

Reply:
It’s not needed. Till 0.7 we have v002 version https://github.com/open-metadata/OpenMetadata/tree/0.7.0/bootstrap/sql/mysql . When users upgrade the bootstrap migrate will apply v003 . So each release if we have any schema changes we can increment this by one. All changes related to that version can belong to one sql file

Reply:
Yes to what @harshach said. I deleted v004 for that reason. We need to pay attention and have one version file per release, if database needs changes.